Documentation/ApiOverview/RequestLifeCycle/Typo3Request.rst

```diff
@@ -84,12 +83,6 @@ it is set.
 ..  literalinclude:: _CodeSnippets/_ViewHelper.php
     :caption: EXT:my_extension/Classes/ViewHelpers/MyViewHelper.php
```

Comment:
I am confused. The file MyLinkViewHelper.php was changed and MyViewHelper.php is used again. Was this intentional? Do we have to update the code snippets? Are the code snippets up to date?